src/Uno.UI/UI/Xaml/Controls/WebView/WebView.cs

```diff
@@ -14,9 +14,13 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Uno.Extensions;
+using Uno;
 
 namespace Windows.UI.Xaml.Controls
 {
+#if __WASM__ || __SKIA__
+	[NotImplemented]
+#endif
```

Comment:
Is this supposed to be included in the docs PR?

Reply:
I changed it as part of the docs work because it sets the content of the [implemented views page](https://platform.uno/docs/articles/implemented-views.html)

and because it ain't implemented on WASM or Skia